gitpod-io/gitpod
# Add RubyMine and WebStorm as IDEs

## Description


Add RubyMine and WebStorm as IDEs 🎉 

## Related Issue(s)

- https://github.com/gitpod-io/gitpod/issues/7751

## How to test

- Open preview env, check IDE options added
- Select RM and WS as IDE, open workspace check if it works
- Open JetBrains Gateway, change Gitpod host to this preview env, and check if IDE options added
- Check also [GH Action](https://github.com/gitpod-io/gitpod/actions/runs/3158620337)

## Release Notes

```release-note
Add `RubyMine` and `WebStorm` IDEs
```

## Documentation

- https://github.com/gitpod-io/website/issues/2832

## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`
